applications/asset_tracker_v2/src/cloud/cloud_codec/nrf_cloud/nrf_cloud_codec.c

```diff
@@ -493,6 +498,9 @@ int cloud_codec_encode_data(struct cloud_codec_data *output,
 		 */
 		int64_t rsrp_ts = modem_dyn_buf->ts;
 
+		/* Adjust RSRP to dBm */
+		modem_dyn_buf->rsrp -= 140;
+
 		len = snprintk(rsrp, sizeof(rsrp), "%d", modem_dyn_buf->rsrp);
 		if ((len < 0) || (len >= sizeof(rsrp))) {
 			LOG_ERR("Cannot convert RSRP value to string, buffer to small");
```

Comment:
should be "buffer too small" in this and the other log msgs.

Reply:
I always confuse "to" and "too" 😢